src/views/ChangePasswordExpired/ChangePasswordExpired.pug

```diff
@@ -0,0 +1,26 @@
+//-
+  Copyright (C) 2019 Kaleidos Open Source SL
+
+  This file is part of Dont Worry Be Happy (DWBH).
+  DWBH is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  DWBH is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with DWBH.  If not, see <https://www.gnu.org/licenses/>.
+
+.expired
+  p.title {{ $t("CHANGEPASS_EXPIRED.MESSAGE") }}
+
+  .link
+    a.login(
+        v-on:click="goToReset()"
+        data-testid="back-to-reset"
+        tabindex="-1"
```

Comment:
Change to negative index, like the other day.

Reply:
Oh yes I forgot. Removed